src/js/07-copy-to-clipboard.js

```diff
@@ -1,7 +1,7 @@
 ;(function () {
   'use strict'
   var runCodeLangs = { cpp: 'cc', csharp: 'dotnet', js: 'nodejs', python: 'py', ruby: 'rb' }
-  var displayLangs = { sqlpp: 'sql++', n1ql: 'sql++' }
+  var displayLangs = { sqlpp: 'sql++' }
```

Comment:
Does this mean that any old docs that still use `n1ql` will display `sqlpp`?

Reply:
I might just be confused as I don't know enough about this area, but maybe @RayOffiah is clearer on this?

Reply:
(N.B. For the avoidance of doubt, this PR is _only_ for code blocks, not for the `{sqlpp}` attribute)

Currently, in old docs that still use `[source,n1ql]`, the code blocks will be labelled in the output as `SQL++`. This is wrong ✖️ 

With this PR, in old docs that still use `[source,n1ql]`, code blocks will be displayed in the output as `N1QL`; and in new docs that use `[source,sqlpp]`, code blocks will be labelled in the output as `SQL++`. This is correct ✔️

Reply:
The `n1ql: 'sql++'` replacement should never have gone in there, it was a mistake